eng/Generate.ps1

```diff
@@ -9,7 +9,6 @@ $ErrorActionPreference = 'Stop'
 $repoRoot = Resolve-Path (Join-Path $PSScriptRoot '..')
 
 $swaggerDefinitions = @{};
-$swaggerTestDefinitions = @{};
```

Comment:
this and all its related contents are removed.
The initial purpose of this is to support the test generation for MPG, which is generated separately from the source code. We have to run autorest in the source code directory to generate the source code, and then run autorest in the test project directory to generate the samples/tests.
In our previous change, the test project in MPG is now generated along with the source code, we no longer need to invoke autorest twice to generate the test project, therefore these are not needed any more.